scripts/setup-test-env.sh

```diff
@@ -15,10 +15,12 @@ case $VERSION in
     VERSION="0.11";;
   1.*)
     VERSION="1.1";;
```

Comment:
maybe add `2.4*)` here
and leave `2.*)` unchanged.

Reply:
What your opinion about keep test matrix in sync with `kafka_protocol`?

- https://github.com/kafka4beam/kafka_protocol/blob/4.0.1/scripts/setup-test-env.sh#L8-L22
- https://github.com/kafka4beam/kafka_protocol/blob/4.0.1/.github/workflows/ci.yml#L12-L27

Reply:
But I did what do you ask: https://github.com/kafka4beam/brod/pull/472/commits/ad5065d97266d9a5a1627b5c17d8c6f57721327f

Reply:
> What your opinion about keep test matrix in sync with kafka_protocol?

that would be greate.

Reply:
Done https://github.com/kafka4beam/brod/pull/472/commits/b0955a47408c4502256c1dd27162e4ae3cba9a6b